src/Rector/AbstractScopeAwareRector.php

```diff
@@ -20,23 +16,12 @@ abstract class AbstractScopeAwareRector extends AbstractRector implements ScopeA
 {
     /**
      * Process Node of matched type with its PHPStan scope
-     * @return Node|Node[]|null|NodeTraverser::*
      */
-    public function refactor(Node $node): int|array|Node|null
+    public function refactor(Node $node)
     {
-        /** @var MutatingScope|null $currentScope */
-        $currentScope = $node->getAttribute(AttributeKey::SCOPE);
-
-        if (! $currentScope instanceof Scope) {
-            $errorMessage = sprintf(
-                'Scope not available on "%s" node, but is required by a refactorWithScope() method of "%s" rule. Fix scope refresh on changed nodes first',
-                $node::class,
-                static::class,
-            );
-
-            throw new ShouldNotHappenException($errorMessage);
-        }
-
-        return $this->refactorWithScope($node, $currentScope);
+        throw new ShouldNotHappenException(sprintf(
+            'The `Rector\Rector\AbstractScopeAwareRector` is removed, use `Rector\Rector\AbstractScopeAwareRector` instead, see upgrading guide %s',
```

Comment:
This message should be updated

Reply:
Updated https://github.com/rectorphp/rector-src/commit/9b5f698ab34b428bf5aedb6c5ff7d0134db2e6f7